### ISSUE TITLE
Installation of VM templates doesn't complete unless cluster preparation is restarted.

### ISSUE THREAD

**Issue by granth7** [nforgeio/neonKUBE#1226]
When using neon-cli's cluster prepare command I noticed one of the hosts got stuck on install: virtual machine template (slow). I thought it was just really slow so waited for about 10 minutes before stopping and restarting the service. After restarting the cli it seems that the vm template was successfully installed and the check proceeded to the next host where the same thing happened. All I had to do was restart the cli a couple times to resolve this issue so not a huge priority! If there's anything I can do to provide more information please let me know.

**Comment by granth7** [comment]
Looks like the real issue was that I was running out of disk space. @marcusbooyah suggested a more detailed message might be useful in this scenario.

**Comment by marcusbooyah** [comment]
Yes, we need to check that the host(s) can fit the cluster before we start building it, and then provide messaging if not/why not.

**Comment by jefflill** [comment]
I think we already have an open issue for this.

—Jeff

**Comment by granth7** [comment]
Yep you're right, this is a duplicate of #994. Gonna close this one then.
